fetch/fetch.go

```diff
@@ -0,0 +1,174 @@
+package fetch
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+var (
+	client *ssm.SSM
+	paths  []string
+	tags   []string
+
+	trueBool = true
```

Comment:
https://github.com/gametimesf/pkg/blob/master/pointer/pointer.go#L9

Reply:
-1 this is a public repo since it was forked, it should not depend on stuff in private repos